src/components/ProfileCard.js

```diff
@@ -32,6 +38,18 @@ const ProfileCard = () => {
     updateUser({ email: userEmail, profile: { ...userProfile, ...profile } })
   }
 
+  // Open / close menu
+
```

Comment:
What is happening in this useState?

Set the new email as email in newsletter?